### PR TITLE
Make host dynamic for DNS records to support records on subdomain

### PR DESCRIPTION
**Pull request by puneet60**

### CODE REVIEW

File: brevo.com.domain-authentication.json [hunk 1]
```diff
@@ -3,31 +3,31 @@
...
-      "host": "%dkim_selector%._domainkey",
+      "host": "%dkim_host%",
       "data": "%dkim_public_key%",
       "ttl": 3600
     },
     {
       "groupId": "dmarc",
       "type": "TXT",
-      "host": "_dmarc",
+      "host": "%dmarc_host%",
```
Comment (pawel-kow) [inline brevo.com.domain-authentication.json:30]: Just so I get the changes right... why would _dmarc ever be on a different host?

Comment (puneet60) [reply brevo.com.domain-authentication.json:30]: We have a use case where we would add dmarc to subdomain.
For example: `_dmarc.abcd` 
In current template we would not able to add dmarc record on subdomain

Comment (pawel-kow) [reply brevo.com.domain-authentication.json:30]: Unterstood. Would in this case all the other records land on the same subdomain?

Comment (puneet60) [reply brevo.com.domain-authentication.json:30]: Yes all record will be on same subdomain, just difference of host.
For example dkim can be `mail._domainkey.abcd` and dmarc will be `_dmarc.abcd`

Comment (pawel-kow) [reply brevo.com.domain-authentication.json:30]: I guess then this template change is not needed at all? Just use `host` parameter when applying the template.

Test yourself here: https://pdnsadmin.revproxy.short-lived.de/dc/free/templateedit

My test:
image

Comment (puneet60) [reply brevo.com.domain-authentication.json:30]: Thanks a lot @pawel-kow 
That's exactly I wanted it to work. Now No further need to change template.
We can close this PR

**Pull request closed without merging**